code/WorkInProgress/cryospawn.dm

```diff
@@ -168,6 +170,11 @@
 			if (H.sims)
 				for (var/name in H.sims.motives)
 					H.sims.affectMotive(name, 100)
+
+		var/datum/db_record/crew_record = data_core.general.find_record("name", L.real_name)
```

Comment:
```suggestion
		var/datum/db_record/crew_record = data_core.general.find_record("id", L.datacore_id)
```
This prevents bugs in case of two people being named the same as datacore_id always points to the exact right record for L.